Changelog

```diff
@@ -1,17 +1,57 @@
-Version 3.3, 2020-XX-XX
+Version 3.3, *2020-XX-XX*
+
+  Features:
+    * New token type: WebAuthn/FIDO2 token (#1468)
+      * WebAuthn hardware tokens can now be used with privacyIDEA
+    * New token type: Indexed Secret. (#1986)
+      * A challenge-response token that asks the user for positions from a list of secrets.
+    * *OPEN New sophisticated logging facility (#1580)*
+      * *TEXT TEXT TEXT*
+
+  Fixes:
+    * Fixed unauthorized statistics view (#1238)
+    * Fixed API endpoints to avoid redirects (#1999)
+    * Fixes a bug which caused an exception during PSKC key file container import (#1915)
+    * Fixed url_decode padding before it could cause any issues (#2000)
+    * Initialize rtype in user_object correctly (#2007)
+    * Fixed an inconsistency of start_tls with postgres SQL (#2025)
+    * Fix wrong type splitting of questionaire token (#2026)
+    * Fixed a bug which could cause missing audit entries when using the 
+      ContainerAudit module (#2029)
+    * Fixed a bug which prevented defining an SQL resolver without a password (#2030)
+    * Fixed missing "position" argument on event import with pi-manage (#2036)
+    * Fixed an issue that lead to failing tests in some cases (#2041)
```

Comment:
I am not sure if this is a change from 3.2 -> 3.3. Or if this simply something that happended during development of 3.3.
So then we should remove it from the changelog, since it is no change from 3.2->3.3

Reply:
ok